app/visualisation_methods/dendrogram/_modal.html.haml

```diff
@@ -50,10 +50,17 @@
 :javascript
   $(function(){
 
+
     /* function from global variable moes_info_json create html checkbox and then replacing exiting with new ones*/
     window.reload_dendrogram_params = function(){
       $('#busy_dendrogram').show();
       var new_checkbox ="";
+
+      if (("#{params[:stand_alone]}" == "true") || ("#{params[:stand_alone]}" == "false" && "#{params[:moes_info_exist]}" == "false")) {
```

Comment:
jaki jest cel w używaniu ``"#{var}"``? Jeśli rzutowanie na string, to trzeba użyć ``.to_s``